net/libfab/libfab.c

```diff
@@ -681,7 +682,7 @@ static void libfab_poller(struct m0_fab__tm *tm)
 		while (tm->ftm_ntm->ntm_state < M0_NET_TM_STARTED)
 			/* No-op */;
 
-		m0_nanosleep(0, NULL);
+		sched_yield();
```

Comment:
Note that this may schedule the thread/process on a different core than was there earlier.